code/modules/clothing/gloves/color.dm

```diff
@@ -191,6 +180,19 @@
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
 	strip_delay = 60
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 70, ACID = 50)
```

Comment:
tbh, I'm not sure why the cap's gloves aren't fireproof. aren't they coated in diamonds?